pandas/core/internals/array_manager.py

```diff
@@ -1287,8 +1287,24 @@ def apply(self, func, **kwargs):
         return type(self)([new_array], self._axes)
 
     def setitem(self, indexer, value):
```

Comment:
This is an existing method, all other Manager methods with general `indexer` argument are not yet typed, so I would prefer to leave that for a typing-specific PR (I also thought that we generally want to avoid adding `Any` for "not yet typed" unless it's really "any").

Reply:
kk